### ISSUE TITLE
Inactive 2FA Report Should Ignore/Separate "Proprietary software based 2FA"

### ISSUE THREAD

**Issue by SethFalco** [bitwarden/server#1122]
When using Bitwarden's "Inactive 2FA Report", it includes websites which don't support normal TOTP applications, but instead proprietary applications.

Examples include:
* SendGrid
* Steam
* TransferWise

![image](https://user-images.githubusercontent.com/22801583/106366636-7d396380-633d-11eb-950c-fde72bdc7531.png)
> For example, Steam can only have TOTP via the Steam mobile application.

I think this is unexpected behavior, and that websites should only appear here if they are have a check mark, not if they only have an i with "Proprietary software based 2FA".
So basically the sites that appear in: https://twofactorauth.org/api/v2/totp.json

Currently Bitwarden is displaying that logins without 2FA was found, when I do already have 2FA on them but just not with Bitwarden because the services don't physically allow it.

---

An alternative could be to make a separate table where these other applications that require proprietary software are displayed. This way users won't have to manually check Two Factor Auth if it can actually be added to Bitwarden.

**Comment by kspearrin** [comment]
FYI, steam is supported in bitwarden TOTP generator. See https://community.bitwarden.com/t/steam-totp-support/988/6

**Comment by Virinum** [comment]
I think this got fixed in bitwarden/web#849

**Comment by SethFalco** [comment]
@Virinum It looks like you're right! \\(^-^)/ 

(For future reference, if you knew about this issue before, you could've done "Closed #1122" to make your PR close this issue automatically.)

I'll close it myself for now, thank you for making the contribution! Once it's released I'll manually check it just in case, if there're any problems, then I'll just open this issue again.

**Comment by Virinum** [comment]
I didn't see this issue here before creating the PR because I just looked at the web issues. :D

Just stumbled upon this issue here and it looked familiar.
